src/Host/Client/Impl/Definitions/PlotDeviceProperties.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.R.Host.Client {
+    public struct PlotDeviceProperties {
+        public int Width { get; }
+        public int Height { get; }
+        public int Resolution { get; }
+
+        public PlotDeviceProperties(int width, int height, int resolution) {
+            Width = width;
+            Height = height;
+            Resolution = resolution;
+        }
+
+        public static PlotDeviceProperties CreateDefault() {
```

Comment:
Since this is a struct, with true copy semantics, this could be a readonly static field.